src/main/java/uk/gov/hmcts/reform/draftstore/config/FlywayConfiguration.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.draftstore.config;
+
+import org.flywaydb.core.Flyway;
+import org.springframework.boot.autoconfigure.AutoConfigureAfter;
+import org.springframework.boot.autoconfigure.AutoConfigureBefore;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.autoconfigure.flyway.FlywayAutoConfiguration;
+import org.springframework.boot.autoconfigure.flyway.FlywayMigrationStrategy;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
+import org.springframework.boot.autoconfigure.orm.jpa.HibernateJpaAutoConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.hmcts.reform.draftstore.data.migration.FlywayNoOpStrategy;
+
+@AutoConfigureAfter({
```

Comment:
These java classes are a copy-paste from send-letter-producer. The purpose is to prevent DB migrations from talking place at application startup.